src/Main.java

```diff
@@ -32,4 +35,28 @@ public void setHeight(double height) {
             this.height = height;
         }
     }
+
+    public static class Square extends Rectangle {
+        protected double side;
```

Comment:
Why protected when we have getter and setter ?

Reply:
and we don't need to define new variable

Reply:
I think we need to define it instead of having a getter and setter for both width and height which should change together all the time